### DISCUSSION TITLE
Unable to use more than 101 LEDs (Noob)

### DISCUSSION THREAD

**Discussion by daz417** [krojew/OpenRGB-Ambient#10] [Q&A]
I am unable to use the Ambient effect for all the LEDs, I've tried to follow the setup instructions, but still, the sliders won't switch on past 101 LEDs, can anyone help me configure the setup? I'm using E1.31 with WLED. The effects tab seems to be working fine with all the LEDs without any issues. Don't know what I'm doing wrong here.

**Comment by krojew** [comment]
Can you select all the LEDs in the main window or do they also cut off at 101?

**Reply by daz417** [reply]
I can select all of them in the main window and can also change the colors individually, but the ambient plugin doesn't seem to work. I've just tried using the Ambient effect, it works but not the standalone one.

**Reply by daz417** [reply]
Thanks for the swift reply.

**Reply by daz417** [reply]
![image](https://github.com/krojew/OpenRGB-Ambient/assets/58841971/874feb36-b349-42d0-8b49-6f0e1667b187)

**Reply by daz417** [reply]
Dunno how, but it started working while I was messing with the ambient effect in the effects tab, turned both of them on, it started flickering, once I turned the effect off, it started working with all the LEDs, for me its a miracle.

**Comment by krojew** [comment]
Are those two separate devices? If so, you need to configure them individually.

**Reply by daz417** [reply]
Sorry for the confusing screenshot, I am using only the second device (WLED screen).

![image](https://github.com/krojew/OpenRGB-Ambient/assets/58841971/692a8d9b-b956-4fa2-a31c-448751dc0392)
